pastis/launchers/run_rst.py

```diff
@@ -0,0 +1,32 @@
+"""
```

Comment:
The CI is failing on some syntax error in this script that we will have to look at.